media-sound/milkytracker/milkytracker-1.02.00.ebuild

```diff
@@ -23,8 +28,10 @@ RDEPEND="
 DEPEND="${RDEPEND}"
 
 PATCHES=(
-	"${FILESDIR}"/${P}-docdir.patch
-	"${FILESDIR}"/${P}-cmake.patch
+	"${DISTDIR}/${P}-cmake.patch"
+	"${FILESDIR}/${P}-useafterfree-fix.patch"
```

Comment:
What does this fix? No CVE?

Reply:
Whoops, filed this PR before the CVE was assigned and forgot to adjust. Will correct it.